EIPS/eip-simplify-161.md

```diff
@@ -0,0 +1,308 @@
+---
+eip:
+title: Simplify EIP 161
+description: Simplify EIP 161 and retroactively deprecate unused aspects of it
+author: Peter Davies (@petertdavies)
+discussions-to:
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-02-02
+requires: 161
+---
+
+
+## Abstract
+
+EIP 161 is overly complex and has a number of edgecases that are poorly documented and tested. This EIP takes advantage of the complete removal of all remaining empty accounts in block 14,049,881 to clarify it and allows implementors to not implement various edgecases that never occured and are not possible in future.
+
+In particular, this EIP permits implementors who do not wish to support historical blocks to not implement state clearing at all.
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+Retroactively replace EIP 161 with the following rules:
```

Comment:
Do you mean these rules apply from the same block EIP-161 did? Perhaps better to make it clear here.